### DISCUSSION TITLE
Espias

### DISCUSSION THREAD

**Discussion by Nikzza** [INGCOM-UNRN-P1/cursada-2023#225] [Preguntas y respuestas] [answered]
Hola Martin, tenia una duda
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141664052/7eea935f-4357-40e1-a8ea-af196a0d5b13)

que significaria c? en la entrada por linea de comandos?

puede ser que signifique cifrar, y si utilizo d es descifrar? es lo unico que se me ocurre

**Comment by martinvilu** [comment, marked as answer]
Si, pero no hay ninguna obligación que tenga esa forma puntualmente.

(Recordá que no es un chat, completá tu pregunta en el espacio apropiado)
